agent/config.go

```diff
@@ -1448,6 +1454,13 @@ func DecodeConfig(r io.Reader) (*Config, error) {
 		}
 		result.DeprecatedHTTPAPIResponseHeaders = nil
 	}
+
+	// Set the ACL replication enable if they set a token, for backwards
+	// compatibility.
+	if len(result.ACLReplicationToken) > 0 {
```

Comment:
why not `if result.ACLReplicationToken != ""` since it is a string? Use of `len` always makes me think of an array (which it technically is)